user_guide/content/modules/about_scenario_driven_planning.adoc

```diff
@@ -0,0 +1,58 @@
+[#about_scenario_driven_planning]
+= About Scenario Driven Planning
+
+One of the development methodologies supported by the Planner is the Scenario driven development approach to plan and track your development work. The Scenario driven development process uses the following work item type hierarchy:
```

Comment:
A lot of published work about various methodologies uses this format for the name: Xyz-driven development (e.g. Test-driven development, Behavior-driven development) so I'd recomment s/Scenario driven development/Scenario-driven development everywhere.

Reply:
+1